torchaudio/datasets/utils.py

```diff
@@ -76,41 +65,85 @@ def makedir_exist_ok(dirpath):
             raise
 
 
-def download_url(url, root, filename=None, md5=None):
-    """Download a file from a url and place it in root.
+def download_url(url, download_folder, hash_value=None, hash_type="sha256"):
```

Comment:
This makes the assumption that we want to download to a folder. Is it possible to split this out?

Reply:
Yes, it can be done.